inspirehep/modules/search/static/js/search/app.js

```diff
@@ -22,9 +22,13 @@
  */
 
 require([
+    'angular',
+    'angular-sanitize',
+    'angular-ui-bootstrap',
```

Comment:
This first 3 are not needed, since `settings.js` already defines the `deps` so they will be correctly loaded.